twitchio/websocket.py

```diff
@@ -516,6 +538,11 @@ def _fetch_futures(self):
 
     async def _close(self):
         self._keeper.cancel()
+        self._task_cleaner.cancel()
+
+        for task in self._background_tasks:
+            task.cancel()
```

Comment:
cancelling the tasks probably isn't the best strategy, as that will dangle the error handlers if they get cancelled.

Reply:
Good point